### ISSUE TITLE
poor pwa score on google lighthouse audit

### ISSUE THREAD

**Issue by touseefbsb** [dhananshan/PWA-Asp.NetCore#3]
I ran the sample app succesfully and notifications are working great as expected but then I tried to audit it with google chrome's developer tools audit tool known as light house which tells us how good our app is in all departments, and I am posting the score here, and as you can see the score suggests its not a good pwa app. You can try it yourself as well :) do you have any idea how this can be improved, there are some suggestions written in the picture as well if they make sense to you.

![audit](https://user-images.githubusercontent.com/11542189/31320470-8a04f714-ac8e-11e7-865c-5403733a6f7f.PNG)

**Comment by dhananshan** [comment]
@touseefbsb  Thanks for info... I did an audit as well before.. some of the failed audits are 
HTTPS: not necessary for a demo ,
Performance: mainly because it reads data from external source (RSS)
but yes definitely we can update the other consideration to get a good score...

**Comment by touseefbsb** [comment]
yeah rss data feed is not required for every app, but other factors are important for pwa :)
